client/test/DOMSupport.res

```diff
@@ -1,3 +1,5 @@
+// WHAT does this do?
```

Comment:
JSdom is a library for testing the DOM when you don't have a real browser. Which is the case in our tests, which run in Node.